vm/devices/get/get_protocol/src/lib.rs

```diff
@@ -1181,12 +1181,8 @@ impl UpdateGenerationId {
 #[bitfield(u64)]
 #[derive(AsBytes, FromBytes, FromZeroes)]
 pub struct SaveGuestVtl2StateFlags {
-    /// Disable nvme_keepalive feature when servicing.
-    #[bits(1)]
-    pub disable_nvme_keepalive: bool,
```

Comment:
I assume this is just really quick fix to quickly unblock the testing and the follow-up is needed.

Reply:
Right. Let's decide as a group what we want the design of this protocol to be, but revert the changes that we know we don't want.